### DISCUSSION TITLE
Option to disable the sidebar transparency

### DISCUSSION THREAD

**Discussion by Flo4604** [scalar/scalar#2728] [Ideas]
![85702](https://github.com/user-attachments/assets/5ff92cb2-1bf6-42cb-a7e9-8e678b5134d3)

I can see the discord images trough this, I would love to have a option for just a nontransparent sidebar

**Comment by hanspagel** [comment]
Oops! Isn’t that a system setting though? I mean we configured it to have transparency, but I think you can disable transparency for all applications on your system.

**Reply by Flo4604** [reply]
Is it though? zed and other apps don't have transparency for me either.

**Comment by cameronrohani** [comment]
We should either get background blur working or remove the transparency, I couldn't get it working last I tried but let's take another look

**Comment by cameronrohani** [comment]
found the solution which was setting vibrancy

https://www.electronjs.org/docs/latest/api/browser-window#winsetvibrancytype-macos

Wasn't happy with the results so most likely going to go back to solid backgrounds in a PR today @Flo4604 🫡

**Reply by cameronrohani** [reply]
No more transparency! :) https://github.com/scalar/scalar/pull/2737
